styles.css

```diff
@@ -0,0 +1,97 @@
+/* Melanie Cascante */
+
+h1 {
+  width: 220px;
+  height: 100px;
+  display: flex;
+  padding-top: 50px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+/* ic_menu */
+
+.ic_menu {
+  padding-top: 15px;
+  padding-left: 60px;
+  background: #fafafa;
+}
+
+/* greeting */
+
+.greeting {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  background: #fff;
+  flex: 1;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
This style might cause some issues in this `section` because you have both an `hx` tag and a `p` tag here 👀  therefore they will both that this `font-weight` which won't match the design.